### DISCUSSION TITLE
[HELP] Cannot typing password sudo for WSL when initialize project using Laravel Sail in Windows 11

### DISCUSSION THREAD

**Discussion by mmuqiitf** [laravel/framework#53255] [Q&A]
### Laravel Version

11.9

### PHP Version

8.3.12

### Database Driver & Version

MySQL 8.0.30 for Windows (Laragon)

### Description

When I want to start a new project using the command:
`curl -s "https://laravel.build/super-apps" | bash`

The installation process went smoothly but when the final adjustment by Laravel Sail needed to enter the sudo password from my WSL Ubuntu, I couldn't type or press the enter key. Just stuck on this process
![image](https://github.com/user-attachments/assets/0c0831d6-2393-4b55-b5fc-4718a6b13edc)

When I canceled the command (when entering password sudo), I couldn't use the command: `./vendor/bin/sail up` to run the laravel apps, but what appeared was the sail code file in my VSCode.
![image](https://github.com/user-attachments/assets/57390ff8-1fc5-452b-a422-d22651277883)

### Steps To Reproduce

1.  Initiate a new project like this : curl -s "https://laravel.build/super-apps" | bash
2. Wait until you should provide password root for WSL
![image](https://github.com/user-attachments/assets/80c9fa00-2fd2-4d70-b82f-8caa5cc83e9d)
3. Then enter the password sudo (but cannot do it just stuck on this process)

**Comment by mmuqiitf** [comment]
I drop this issue on https://github.com/laravel/sail/issues/739 but no one replying to me, i hope in this repo someone can help me

**Comment by flavio-schoute** [comment]
You do need Laravel Sail to be in your WSL environment.

**Reply by chrisonoo** [reply]
I came here with the same problem but you made me realize the solution.

1. https://laravel.com/docs/11.x/installation#sail-on-windows

![image](https://github.com/user-attachments/assets/45903a62-c00e-44a4-90c6-185c0b9e84e0)

2. Docker Desktop > Settings > Resources > WSL integration

![image](https://github.com/user-attachments/assets/2ff76f94-1f64-4379-adff-de35934d38f9)

3. Then we run all commands by going to the appropriate folder in the WSL console but on the local disk (not WSL). Local disks are mounted in the path `/mnt/...` e.g. `/mnt/c` is the C:\ drive

4. This way everything works, although on my computer it is definitely slower than, for example, with xampp.

![image](https://github.com/user-attachments/assets/980b2030-b92e-4fdd-be9b-69a01b7c48af)
